bool[] cal_mask

float64[] lambda

float64[] lambda_high

---

string message

bool success
